activestorage/lib/active_storage/engine.rb

```diff
@@ -18,6 +18,14 @@ class Engine < Rails::Engine # :nodoc:
     config.active_storage.analyzers = [ ActiveStorage::Analyzer::ImageAnalyzer, ActiveStorage::Analyzer::VideoAnalyzer ]
     config.active_storage.paths = ActiveSupport::OrderedOptions.new
     config.active_storage.variable_content_types = [ "image/png", "image/gif", "image/jpg", "image/jpeg", "image/vnd.adobe.photoshop" ]
+    config.active_storage.content_types_to_render_as_binary = [ "text/html",
+                                                                "text/javascript",
+                                                                "image/svg+xml",
+                                                                "application/postscript",
+                                                                "application/x-shockwave-flash",
+                                                                "text/xml",
+                                                                "application/xml",
+                                                                "application/xhtml+xml" ]
```

Comment:
Style nit: indent the array elements one level under the assignment.

```ruby
config.active_storage.content_types_to_render_as_binary = [
  "text/html",
  "text/javascript",
  # ...
]
```

Reply:
Sure thing! The truth is, I prefer one level under the assignment too! But I wasn't sure which one would be preferred, so I searched in the codebase and found this example: 

https://github.com/rails/rails/blob/5a5014688873f1d6e1b66075eea8a4356b5a4d07/railties/lib/rails/tasks/tmp.rake#L7-L10

And did the same. But it's true in that example the indentation is much smaller, so it's not really applicable directly 😅